### PR TITLE
MFT: fix macros compilation issues

### PR DESCRIPTION
**Pull request by bovulpes**
- Add a new directory in the MFT directory with macros for internal use
- Move pixel segmentation from macro/run_sim_mft.C to Detector::Initialize()
- Remove obsolete files

### CODE REVIEW

File: Detectors/ITSMFT/MFT/macros/SetIncludePath.C [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (sawenzel) [inline Detectors/ITSMFT/MFT/macros/SetIncludePath.C]: Can you please avoid an anonymous macro here and make a function `void SetIncludePath()`. Unfortunately, on MacOs, interpreting this macro returns a nonzero exit code which our unit test considers a failure. We had fixed this in the original macro directory.

File: Detectors/ITSMFT/MFT/macros/rootlogon.C [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (sawenzel) [inline Detectors/ITSMFT/MFT/macros/rootlogon.C]: why is this necessary? ROOT6 finds libraries automatically.

Comment (bovulpes) [reply Detectors/ITSMFT/MFT/macros/rootlogon.C]: It is just to load the macro SetIncludePath, I will remove the libraries loading.

Comment (bovulpes) [reply Detectors/ITSMFT/MFT/macros/rootlogon.C]: Why do I have this error if the libraries are automatically loaded:

run_sim_mft_C.so: undefined symbol: _ZN2o23MFT8DetectorC1Ev

Comment (sawenzel) [reply Detectors/ITSMFT/MFT/macros/rootlogon.C]: Good question. I just quoted what the ROOT team told me ... So just leave it for the moment.
